src/main.py

```diff
@@ -467,6 +459,28 @@ def metric_page():
         
         st.plotly_chart(fig7, theme="streamlit")
 
+    if model_name in ["pvnet_v2", "National_xg"]:
```

Comment:
In terms of UX, there's no indication that the pinball and exeedance charts are there unless you know about them. maybe add a notification under the model selection that they are available for `pvnet_v2` and `national_xg`?

Reply:
hmm, tempted to say we dont anything. Just have knowledge that cnn doesn't have probablistic forecasts